plugins/modules/aws_glue_job.py

```diff
@@ -337,10 +370,12 @@ def main():
         dict(
             allocated_capacity=dict(type='int'),
             command_name=dict(type='str', default='glueetl'),
+            command_python_version=dict(type='str', default='2'),
             command_script_location=dict(type='str'),
             connections=dict(type='list', elements='str'),
             default_arguments=dict(type='dict'),
             description=dict(type='str'),
+            glue_version=dict(type='str', default='0.9'),
```

Comment:
```suggestion
            glue_version=dict(type='str'),
```

Reply:
Fair point. My default was based on boto3 documentation for [create-job](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.create_job):

> Jobs that are created without specifying a Glue version default to Glue 0.9.

I committed the recommended changes.